Tracking/CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ project(${PackageName})
 find_package(MarlinUtil REQUIRED)
 list(APPEND CMAKE_MODULE_PATH $ENV{PANDORAPFA}/cmakemodules)
 find_package(PandoraSDK REQUIRED)
-FOREACH( pkg  MarlinUtil PandoraSDK )
+find_package(Delphes REQUIRED)
+FOREACH( pkg  MarlinUtil PandoraSDK Delphes )
```

Comment:
This is not needed, since the delphes LIBRARIES and INCLUDE directories are added to the specific targets?